docs/syntax/scss.md

```diff
@@ -278,7 +278,23 @@ Modules that make use of styles defined in other modules that support silent mod
 		@include oAnotherThingFoo();
 	}
 
-Finally, in documentation, modules *must* provide information about both silent and non-silent methods, where supported, and must put the default first (i.e. if silent mode is by default **on**, the module must document the silent mode integration first).
+When a module that supports silent mode is used with silent mode turned off (i.e. `$o-{modulename}-is-silent: false`), after the module has output the CSS, it *must* also reset the value of `$o-{modulename}-is-silent` to true, (this prevents the accidental output of styles if the module is included a second time):
```

Comment:
Good point, I'll look at making that clearer this morning. 

On you second point, we do have that in the spec about Origami modules and that's followed (as far as I'm aware). However, the issue has come up in places like Next where they have a lot of smaller components that might each include the same Origami module as a dependency, and if the first switches silent mode to `false` the latter then might output the component in `silent-mode:false` as well, duplicating the styles. We've had a couple of modules PR'd to reset the `$o-{modulename}-is-silent` variable to `true` so making this part of the spec and checking that every module is consistent seemed sensible.